### PR TITLE
AUT-2393: Axios upgrade from 0.25.0 to 0.1.6.8

### PR DESCRIPTION
**Pull request by ayoshebby**
## What

Bumps [axios](https://github.com/axios/axios) from 0.25.0 to 1.6.8.

### Release notes:
**Bug Fixes**

- AxiosHeaders: fix AxiosHeaders conversion to an object during config merging (https://github.com/axios/axios/issues/6243) ([2656612](https://github.com/axios/axios/commit/2656612bc10fe2757e9832b708ed773ab340b5cb))
- import: use named export for EventEmitter; ([7320430](https://github.com/axios/axios/commit/7320430aef2e1ba2b89488a0eaf42681165498b1))
- vulnerability: update follow-redirects to 1.15.6 (https://github.com/axios/axios/issues/6300) ([8786e0f](https://github.com/axios/axios/commit/8786e0ff55a8c68d4ca989801ad26df924042e27))

### CODE REVIEW

File: src/assets/javascript/cookies.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ var cookies = function (trackingId, analyticsCookieDomain) {
       setCookie(
         COOKIES_HISTORY_LENGTH,
         window.history.length
-        );
+      );
```
Comment (BeckaL) [inline src/assets/javascript/cookies.js:73]: could we get rid of these unrelated changes from this commit?

File: src/utils/http.ts [hunk 2]
```diff
@@ -97,7 +97,7 @@ export class Http {
       apiError = new ApiError(
         error.message,
         error.response.status,
-        error.response.data
+        error.response.data as string
```
Comment (BeckaL) [inline src/utils/http.ts:100]: Could you add some info into the commit message as to why we need this change? Is this an axios bump thing?

**Review by BeckaL — COMMENTED** [review]
Could we reword the commit to remove the WIP comment? Also, what's our rollout plan, are we going to pause the pipeline?

File: package.json [hunk 3]
```diff
@@ -65,7 +65,7 @@
     "@otplib/core": "^12.0.1",
     "@otplib/plugin-base32-enc-dec": "^12.0.1",
     "aws-sdk": "^2.1356.0",
-    "axios": "^0.25.0",
+    "axios": "^0.28.0",
```
Comment (mattvot) [inline package.json:68]: Is it possible to upgrade this to the latest `axios` version, currently `1.6.8`?

File: src/utils/http.ts (hunk 2 above)
Comment (ayoshebby) [reply src/utils/http.ts]: There was a failure in the pre-merge checks regarding of error on that line:

`Argument of type 'unknown' is not assignable to parameter of type 'string'.
`
Only seems to come up when axios is upgraded. 

I've now bumped the version to 1.6.8, and that error no longer exists

File: package.json (hunk 3 above)
Comment (ayoshebby) [reply package.json]: I've updated the version to 1.6.8 now, looks good asides from a change required in the https.ts file where it was enforcing to implement the full AxiosHeader properties

**Review by mattvot — COMMENTED** [review]
Looking good. Just trying to assess if any more changes are required / we're not going to have hidden issues around. I've one suggestion for the moment.

**Review by mattvot — APPROVED** [review]
Just ran through some smoke testing and seen the flows work when targeting the build env. Approved.

**Review by mattvot — APPROVED** [review]

**Review by BeckaL — APPROVED** [review]
